frontend/app/services/EventbriteService.scala

```diff
@@ -207,6 +227,10 @@ object EventbriteService {
     GuardianLiveEventService.getPreviewEvent(id)
   }
 
+  def getPreviewDiscoverEvent(id: String): Future[RichEvent] = Cache.getOrElse[Future[RichEvent]](s"preview-event-$id", 2) {
+    DiscoverEventService.getPreviewEvent(id)
+  }
+
   def getPreviewMasterclass(id: String): Future[RichEvent] = Cache.getOrElse[Future[RichEvent]](s"preview-event-$id", 2) {
```

Comment:
I will take this out for the moment and do this as a separate pull request when adding in discover events to the event-overview tool